tests/Feature/SimpleElectionTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Tests\Feature;
+
+use Tests\TestCase;
+use Illuminate\Foundation\Testing\WithoutMiddleware;
+use Illuminate\Foundation\Testing\DatabaseMigrations;
+use Illuminate\Foundation\Testing\DatabaseTransactions;
+
+class SimpleElectionTest extends TestCase
+{
+    use DatabaseMigrations;
+    use WithoutMiddleware;
```

Comment:
This trait disables middleware like CSRF protection that would complicate the tests.